core/cap-0035.md

```diff
@@ -624,6 +627,12 @@ A new operation, `SetTrustLineFlagsOp`, is proposed in this cap to make
 trustline flag modification simpler. Additionally the `AllowTrustOp` operation is
 semantically intended to change authorization and clawback is outside it's scope.
 
+### Set TrustLine Flags Operation
+The `SetTrustLineFlagsOp` will make it easier for issuers to modify trustline flags.

```

Comment:
I think the detail that you're missing is that this is the only reasonable way for an issuer to achieve the goal of almost all accounts clawback enabled with just a few exceptions, because of the way that the clawback flag is inherited by trust lines.

Reply:
Added that detail.